WarehouseManagement.Core/DTOs/Delivery/DeliveryHistoryDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using WarehouseManagement.Common.Enums;
+
+namespace WarehouseManagement.Core.DTOs.Delivery;
+
+public class DeliveryHistoryDto
+{
+    public int Id { get; set; }
+
+    public ICollection<Change> Changes { get; set; } = new HashSet<Change>();
+
+    public class Change
```

Comment:
now that i think about it, lets also add the date when the change has happened.

Reply:
Done :)